packages/ndla-ui/src/Footer/Footer.tsx

```diff
@@ -126,35 +32,59 @@ type Props = {
     url: string;
     label: string;
   }[];
-  languageSelector?: ReactNode;
   auth?: ReactNode;
 };
 
-const Footer = ({ children, commonLinks, links, languageSelector, auth, privacyLinks }: Props) => {
+const StyledColumns = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: ${spacing.normal};
+`;
+
+const StyledHeading = styled(Heading)`
+  text-align: center;
+`;
+
+const StyledDiv = styled.div`
+  color: ${colors.white};
+  position: relative;
+  background: ${colors.brand.dark};
+  overflow: hidden;
+  z-index: ${stackOrder.base};
+`;
+
+const StyledOneColumn = styled(OneColumn)`
+  z-index: ${stackOrder.offsetSingle};
+  position: relative;
+  padding: ${spacing.large} ${spacing.large} ${spacing.xlarge};
+  max-width: 1196px;
```

Comment:
Vil gjerne ha denne et annet sted men vet ikke helt hvor. Den trengs for å overskrive bredden og ha riktig størrelse for skallering. Forslag?

Reply:
Ser vi har en del max-width spredt rundt i koden. Hadde sikkert vore lurt å ta en gjennomgang av det etter kvart. Det kan sikkert ligge her inntil videre.